client/src/i18n/en/grade.json

```diff
@@ -2,20 +2,20 @@
   "GRADE" : {
     "ADD_GRADE" : "Add a Grade",
     "ALL_GRADES" : "All Grades",
-    "CREATED" : "Grade created with success",
+    "CREATED" : "Grade created succcessfully",
     "CHANGE" : "Change Grade",
-    "DELETED" : "Grade deleted with success",
+    "DELETED" : "Grade deleted succcessfully",
```

Comment:
Looks like these successes have one to many `c`s.

Reply:
Looks like you cccaught me copying and pasting!